enzy_htp/_interface/pymol_interface.py

```diff
@@ -101,6 +101,86 @@ def select_pymol_obj(self, pattern: str, pymol_obj_name: str,
 
         return sorted(result)
 
+    def point_mutate(self, pos_key: Tuple[str, int],
```

Comment:
export_enzy_htp_stru() could be a better name so that it is symmetrical with load_enzy_htp_stru()